documentation/assemblies/managing/assembly-rolling-updates.adoc

```diff
@@ -0,0 +1,29 @@
+// This assembly is included in the following assemblies:
+//
+// using/assembly-management-tasks.adoc
+
+[id='assembly-rolling-updates-{context}']
+= Performing manual rolling updates of Kafka and ZooKeeper clusters
```

Comment:
At least for me as a non-native speaker this is confusing. The _Perform manual rolling updates_ I would expect a guide how I can do them manually. But instead, the guide is about how to trigger them only. So maybe we can call it. _Trigger manual rolling updates ..._ or _Request ..._ or _Start ..._.

Reply:
_Manually starting_?